### PR TITLE
added Smart Homes article and modified contributing.md 

### PR DESCRIPTION
**Pull request by chennakeshava1998**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ date: 2017-05-14 23:32:44
 image: '/assets/img/'
 description: 'Introductory post to IEEE NITK'
 tags:
-- IEEE NITK 
+- IEEE NITK
 - CompSoc
 categories:
 - IEEE
```
Comment (salman-bhai) [inline CONTRIBUTING.md]: Put CompSoc/Diode/Piston here instead of IEEE

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -152,6 +152,9 @@ Configuration file: none
 
 ## Step 4: Publishing your article
 
+Create a new branch in your local repo by the command `git checkout -b <branch_name>`
+
+
 Commit and push your changes to your forked repo. Once those changes are in, go ahead and make a pull request to the [main blog's repo](https://github.com/IEEE-NITK/ieee-nitk.github.io).
```
Comment (salman-bhai) [inline CONTRIBUTING.md]: Remove the word called `forked repo`

File: CONTRIBUTING.md (hunk 1 above)
Comment (chennakeshava1998) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 2 above)
Comment (chennakeshava1998) [reply CONTRIBUTING.md]: done

**Comment by chennakeshava1998** [comment]
fixed the problem with viewing the article and authors

**Comment by salman-bhai** [comment]
Looks fine!

**Comment by adithyabhatkajake** [comment]
Merged.

**Pull request merged**